Userland/Libraries/LibC/CMakeLists.txt

```diff
@@ -130,9 +143,8 @@ set(SOURCES ${LIBC_SOURCES} ${AK_SOURCES} ${ELF_SOURCES} ${ASM_SOURCES})
 set_source_files_properties(stdio.cpp PROPERTIES COMPILE_FLAGS "-fno-builtin-fputc -fno-builtin-fputs -fno-builtin-fwrite")
 
 add_library(LibCStaticWithoutDeps STATIC ${SOURCES})
-target_link_libraries(LibCStaticWithoutDeps ssp LibTimeZone)
+target_link_libraries(LibCStaticWithoutDeps PUBLIC ssp LibTimeZone PRIVATE NoCoverage)
 add_dependencies(LibCStaticWithoutDeps LibM LibSystem LibUBSanitizer)
-
```

Comment:
empty line should probably stay

Reply:
Fair. I reworked this commit to put the NoCoverage interface lib in Userland/CMakeLists.txt instead for clarity that it's used in a lot of runtime lib places as well